cmd/controller/main.go

```diff
@@ -41,7 +44,20 @@ import (
 
 func main() {
 	flag.IntVar(&controller.DefaultThreadsPerController, "threads-per-controller", controller.DefaultThreadsPerController, "Threads (goroutines) to create per controller")
-	namespace := flag.String("namespace", "", "Namespace to restrict informer to. Optional, defaults to all namespaces.")
+	namespaceList := flag.String("namespace", "", "Comma-separated list of namespaces to restrict informer to. Optional, if empty defaults to all namespaces.")
+	flag.Parse()
```

Comment:
Is moving this call here going to cause issues when calling `cfg := injection.ParseAndGetRESTConfigOrDie()` later on? I don't _think_ it will, but we should verify.